commit c468e383efe5dcc35468f0eb8d5743dfd0b621bd (refs/changes/67/3167/2)
Author: David Pacheco <dap@joyent.com>
Date:   2018-01-08T10:18:44-08:00 (1 year, 9 months ago)
    
    MORAY-454 pull in node-fast#15
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index f40c3b6..05129e7 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
         "crc": "0.2.1",
         "dtrace-provider": "~0.8",
         "deep-equal": "0.0.0",
-        "fast": "2.3.1",
+        "fast": "2.3.2",
         "ip6addr": "0.1.1",
         "jsprim": "1.4.0",
         "kang": "1.2.0",
